ui/src/context/AnnotationStore.ts

```diff
@@ -4,12 +4,9 @@ import { v4 as uuidv4 } from 'uuid';
 import { Bounds } from './PDFStore';
 import { Label } from '../api';
 
-export class TokenId {
-    constructor(public readonly pageIndex: number, public readonly tokenIndex: number) {}
-
-    toString() {
-        return [this.pageIndex.toString(), this.tokenIndex.toString()].join('-');
-    }
+export interface TokenId {
+    pageIndex: number;
+    tokenIndex: number;
```

Comment:
The class wasn't always being used. This meant sometimes the `tokens` property of the `Annotation` class was an array of `TokenId` instances, and other times it was an array of objects. I think this was the case when loading save annotations, as I imagine we weren't traversing the returned list and converting it into `TokenId` instances.

That said the `toString()` method doesn't appear to be used, so I just reduced it to an interface for simplicity's sake.